src/Parser/Warlock/Destruction/Modules/Talents/SoulHarvestTalent.js

```diff
@@ -21,18 +21,19 @@ class SoulHarvestTalent extends Analyzer {
   }
 
   statistic() {
-    const bonusDmg = this.soulHarvest.talentBonusDmg;
+    const bonusDmg = this.soulHarvest.talentBonusDmg;    
+    const uptime = this.combatants.selected.getBuffUptime(SPELLS.SOUL_HARVEST_TALENT.id) / this.owner.fightDuration;
     return (
-      <StatisticBox
+      <SmallStatisticBox
         icon={<SpellIcon id={SPELLS.SOUL_HARVEST_TALENT.id} />}
-        value={`${formatNumber(bonusDmg / this.owner.fightDuration * 1000)} DPS`}
-        label="Damage contributed"
+        value={`${formatPercentage(uptime)} %`}
+        label="Soul Harvest Uptime"
```

Comment:
Same story as Eradication, we've preferred damage contribution over uptime, to make talent effects comparable.

Reply:
dmg is still there